website/addons/mendeley/model.py

```diff
@@ -133,12 +133,37 @@ def _citation_for_mendeley_document(self, document):
         :return Citation:
         """
         csl = {
-            'id': document.json.get('id'),
-            'type': document.json.get('type'),
+            'id': document.json.get('id')
         }
 
-        if document.title:
-            csl['title'] = document.title
+        CSL_TYPE_MAP = {
+            'book_section': 'chapter',
+            'case': 'legal_case',
+            'computer_program': 'article',
+            'conference_proceedings': 'paper-conference',
+            'encyclopedia_article': 'entry-encyclopedia',
+            'film': 'motion_picture',
+            'generic': 'article',
+            'hearing': 'speech',
+            'journal': 'article-journal',
+            'magazine_article': 'article-magazine',
+            'newspaper_article': 'article-newspaper',
+            'statute': 'legislation',
+            'television_broadcast': 'broadcast',
+            'web_page': 'webpage',
+            'working_paper': 'report'
+        }
+
+        csl_type = document.json.get('type')
+
+        if csl_type in CSL_TYPE_MAP:
+            csl['type'] = CSL_TYPE_MAP[csl_type]
```

Comment:
OK, thanks for the clarification. Is the `type` entry required? Does it need a default?